db/migrate/20170831164445_create_exports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateExports < ActiveRecord::Migration[5.1]
+  def change
+    create_table :exports do |t|
+      t.references :snap_application
+      t.string :destination
```

Comment:
did you think about using active record enum for this like I suggested previously? Would make the code work the same as it already does but this would be an integer field

Reply:
In the past I've been bitten in the ass by changing the available options on an active_record enum fields. (i.e. given `:a, :b, :d` as options, when you add `:c` things that were reading as `:d` are now reading as `:c`) 

This is *probably* fixed by now, but I'm still ActiveRecord::Enum shy.